README.md

```diff
@@ -7,21 +7,21 @@ MIVisionX toolkit is a set of comprehensive computer vision and machine intellig
 
 * [AMD OpenVX](#amd-openvx)
 * [AMD OpenVX Extensions](#amd-openvx-extensions)
-  * [Loom 360 Video Stitch Library](amd_openvx_extensions/amd_loomsl)
-  * [Neural Net Library](amd_openvx_extensions/amd_nn#openvx-neural-network-extension-library-vx_nn)
-  * [OpenCV Extension](amd_openvx_extensions/amd_opencv#amd-opencv-extension)
-  * [RPP Extension](amd_openvx_extensions/amd_rpp)
-  * [WinML Extension](amd_openvx_extensions/amd_winml#amd-winml-extension)
+  + [Loom 360 Video Stitch Library](amd_openvx_extensions/amd_loomsl)
+  + [Neural Net Library](amd_openvx_extensions/amd_nn#openvx-neural-network-extension-library-vx_nn)
+  + [OpenCV Extension](amd_openvx_extensions/amd_opencv#amd-opencv-extension)
+  + [RPP Extension](amd_openvx_extensions/amd_rpp)
+  + [WinML Extension](amd_openvx_extensions/amd_winml#amd-winml-extension)
 * [Applications](#applications)
 * [Neural Net Model Compiler & Optimizer](#neural-net-model-compiler--optimizer)
 * [RALI](#rali)
 * [Samples](samples#samples)
 * [Toolkit](#toolkit)
 * [Utilities](#utilities)
-  * [Inference Generator](utilities/inference_generator#inference-generator)
-  * [Loom Shell](utilities/loom_shell#radeon-loomshell)
-  * [RunCL](utilities/runcl#amd-runcl)
-  * [RunVX](utilities/runvx#amd-runvx)
+  + [Inference Generator](utilities/inference_generator#inference-generator)
+  + [Loom Shell](utilities/loom_shell#radeon-loomshell)
+  + [RunCL](utilities/runcl#amd-runcl)
+  + [RunVX](utilities/runvx#amd-runvx)
```

Comment:
Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)